### DISCUSSION TITLE
Mirasvit Advanced Search

### DISCUSSION THREAD

**Discussion by Davelaar** [breezefront/community#75] [1. Help]
I recently installed Mirasvit Search Again (after upgrading magento to 2.4.6) But my search bar is not working as expected.
I received warning in backend from mirasvit module with Ajax search from swissup. Removed Ajax Search by replacing it in composer. Still no working search bar (it is finding products, but not showing them on autocomplete). Should the integration be updated? Using Search Ultimate v. 2.1.0

**Comment by vovayatsyuk** [comment]
We have an integration with [mirasvit/search-ultimate](https://mirasvit.com/magento-2-extensions/sphinx-search-ultimate.html) v.1.2: https://github.com/breezefront/module-breeze-mirasvit-search-ultimate

Did you install our integration module?

**Comment by roma84** [comment]
I just created a new integration module release. It should work fine with the latest Mirasvit module version.

**Comment by Davelaar** [comment]
Now it is working but some other problems are new now.. will create a new ticket.
